deploy-bridge/.env

```diff
@@ -0,0 +1,8 @@
+SUB_CHAIN_RECEIPIENT=0xd43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d
+SUB_CHAIN_RECEIPIENTSS58=5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY
+ETH_ENDPOINT=https://kovan.infura.io/v3/e8b4790b8e4049cca3c04f738cfa25f2
+SUB_ENDPOINT=ws://parachain:9944/
+ethAddress0=0xa5d3f778d0615977cdd7d12c05aaff34e4b3474b
+ethPrivateKey0=0x8942a0094dfda5c7d40db448588ffb58513802f7c48edaded7bf6157f026567d
+ethAddress1=0x47b9c50c24891bff382004ac73a9e53a230a2802
+ethPrivateKey1=0x62e5c324be3b4a5dc690d66f43f033bb048a3ffb645a8973cb45b436698e27cd
```

Comment:
why do we need two eth keys?
Also, please remove the .env file from the repo to hide the sensitive information, and cover required env variables in documentation instead.

Reply:
Two keys are required because the first derived account in used in Tests as a user.

Reply:
Do you want me to remove all .env files and use exported variables everywhere or just for these two private keys?

Reply:
Ideally not have them at all, and force the user to set all the variables for the deployment. Obviously, examples are fine. Thus probably the best way to move on is to have .env.example in the repo, and .env excluded in gitignore